ranger/README.md

```diff
@@ -8,13 +8,13 @@ This initialization action installs [Apache Ranger](https://ranger.apache.org/)
 You can use this initialization action to create a new Dataproc cluster with Apache Ranger installed:
 
 1. Use the `gcloud` command to create a new cluster with this initialization action. 
-The following command will create a new standard cluster named `<CLUSTER_NAME>`.
+The following command will create a new standard cluster named `<CLUSTER_NAME>` with the Ranger Policy Manager accessible via admin account and `<YOUR_PASSWORD`.
```

Comment:
I was confused about what the user name is, it is better to replace "admin account" with "user \`admin\`".